### ISSUE TITLE
Suggestion/Idea - Text and Item Randomizer

### ISSUE THREAD

**Issue by Nathidraws** [MrCheeze/paper-mario-randomizer#2]
Hi again. _(This isn't an issue, but there doesn't seem to be a way to make suggestions or comments elsewhere.)_

I believe this mod randomizes items, But what if there was another mod that **randomized everything characters said**.

And I also thought a randomizer that **only randomizes Items** would be nice and fun to play through.

Someone has done a similar thing with Ocarina of Time, which was hilarious to play:
https://ootrandomizer.com/ootr.html

Not saying you have to do this, infact I'm sure you've probably thought of them too.
I'm curious how difficult they would be to make.

**Comment by MrCheeze** [comment]
I agree that would be a good idea. But, this was really more of a weekend poc that turned out pretty poorly, so it would probably be better for someone to start from scratch with a more systematic approach before doing that.

**Comment by Nathidraws** [comment]
Thats interesting. 
Btw I think you did a good job on this.
